compiler/src/dmd/astenums.d

```diff
@@ -439,3 +439,22 @@ enum FileType : ubyte
     ddoc, /// Ddoc documentation file (.dd)
     c,    /// C source file
 }
+
+extern (C++) struct structalign_t
```

Comment:
The increasingly inaccurately named AST _enums_ module.

Reply:
I know. But how I remove technical debt is start at the leaves and remove things that don't belong. Those have to go somewhere, but if there isn't an obvious place, it can be dealt with later. Nothing can really be fixed without fixing the leaves. This particular change enabled removing the import of globals.